frontend/public/components/overview/pods-overview.tsx

```diff
@@ -128,22 +130,25 @@ const PodsOverviewList: React.SFC<PodOverviewListProps> = ({ pods }) => (
 
 PodsOverviewList.displayName = 'PodsOverviewList';
 
+const isComplete = (build: K8sResourceKind) => build.status?.completionTimestamp;
+
 export const PodsOverviewContent: React.SFC<PodsOverviewContentProps> = ({
   obj,
   pods,
   loaded,
   loadError,
   allPodsLink,
   emptyText,
-  hasBuildConfig,
+  buildConfigData,
 }) => {
   const {
     metadata: { name, namespace },
   } = obj;
-
+  const { t } = useTranslation();
   const [showWaitingPods, setShowWaitingPods] = React.useState(false);
   const showWaitingForBuildAlert =
-    hasBuildConfig && isDeploymentGeneratedByWebConsole(obj) && pods.some(isPodWithoutImageId);
+    !buildConfigData?.buildConfigs?.[0].builds.some((build) => isComplete(build)) &&
```

Comment:
What if there are no buildConfigs in the returns object?

Reply:
added check for buildConfigs